motoman_gp20_support/urdf/gp20_hl_macro.xacro

```diff
@@ -0,0 +1,173 @@
+        </geometry>
+        <xacro:material_yaskawa_blue/>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://motoman_gp20_support/meshes/gp20_hl/collision/link_1_s.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_2_l">
+      <visual>
+        <geometry>
+          <mesh filename="package://motoman_gp20_support/meshes/gp20_hl/visual/link_2_l.stl"/>
+        </geometry>
+        <xacro:material_yaskawa_blue/>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://motoman_gp20_support/meshes/gp20_hl/collision/link_2_l.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_3_u">
+      <visual>
+        <geometry>
+          <mesh filename="package://motoman_gp20_support/meshes/gp20_hl/visual/link_3_u.stl"/>
+        </geometry>
+        <xacro:material_yaskawa_blue/>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://motoman_gp20_support/meshes/gp20_hl/collision/link_3_u.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_4_r">
+      <visual>
+        <geometry>
+          <mesh filename="package://motoman_gp20_support/meshes/gp20_hl/visual/link_4_r.stl"/>
+        </geometry>
+        <xacro:material_yaskawa_blue/>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://motoman_gp20_support/meshes/gp20_hl/collision/link_4_r.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_5_b">
+      <visual>
+        <geometry>
+          <mesh filename="package://motoman_gp20_support/meshes/gp20_hl/visual/link_5_b.stl"/>
+        </geometry>
+        <xacro:material_yaskawa_blue/>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://motoman_gp20_support/meshes/gp20_hl/collision/link_5_b.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_6_t">
+      <visual>
+        <geometry>
+          <mesh filename="package://motoman_gp20_support/meshes/gp20_hl/visual/link_6_t.stl"/>
+        </geometry>
+        <xacro:material_yaskawa_blue/>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://motoman_gp20_support/meshes/gp20_hl/collision/link_6_t.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <!-- end of link list -->
+
+    <!-- joint list -->
+    <!-- Joints are specified such that the driver rotates the joints about the positive Z axis. -->
+    <joint name="${prefix}joint_1_s" type="revolute">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}link_1_s"/>
+      <origin xyz="0 0 0.540" rpy="0 0 0" />
+      <axis xyz="0 0 1" />
+      <limit lower="-${radians(180)}" upper="${radians(180)}" effort="212.66" velocity="${radians(125)}"/>
```

Comment:
All of the velocity limits in this file appear to be different from the model in #604.

@EricMarcil: would you have any idea why that could be?

Reply:
@gavanderhoorn  No, I do not. I've double checked the values on #604 with two separate sources: the manual and the parameter data from MotoSim, and the values on #604 are the correct ones.

Reply:
Thank you both @acbuynak and @EricMarcil.

@acbuynak: thank *you* for submitting the PR.

Given the state of things I'll close this one now.